os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -133,6 +134,129 @@ extern sq_queue_t g_freemsg_list;
  * Private Functions
  ****************************************************************************/
 
+#ifdef CONFIG_DEBUG_DISPLAY_SYMBOL
+/****************************************************************************
+ * Name: print_dumpstack_symbol
+ * Below API works if there is existance of System.map file in smart fs
+ ****************************************************************************/
+void print_dumpstack_symbol(unsigned long search_addr)
+{
+	FILE *pFile;
+	unsigned long addr;
+	unsigned long next_addr;
+	int first = 0;
+	int last;
+	int mid;
+	char line[128] = { '\0' };
+	char data[6][128] = { {'\0', '\0'} };
+	char c;
+	int word;
+	int ch_in_word;
+	int read_line;
+	int ch_in_line;
+
+	pFile = fopen("/mnt/System.map", "r");
+
+	/* Check if file System.map exists */
+	if (pFile == NULL) {
+		lldbg("Could not open file: /rom/System.map\n");
```

Comment:
Please change `/rom/System.map` to `/mnt/System.map`. It is much easier to understand.